### ISSUE TITLE
Makefile not detecting .mm files in src

### ISSUE THREAD

**Issue by Qix-** [openframeworks/openFrameworks#4468]
I have a few .cpp and a few .mm files in my `src/` directory. While the .cpp files are being recognized and built, the .mm files are not. I'm kind of stumped here, as I've commented out the rule that builds .cpp files from `$OF_ROOT/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk`, which does indeed stop building the .cpp files. But the rule, literally two or so rules down from it, that looks for .mm files doesn't seem to get hit.

Am I missing something here?

**Comment by Qix-** [comment]
Using `make -n` I can see it's building other .mm files in the OF core:

```
echo "Compiling" /src/openframeworks.openframeworks/libs/openFrameworks/video/ofQTKitPlayer.mm
```

But not my project's source files.

**Comment by Qix-** [comment]
This was a bug. You had all the rules for actually *compiling* objective-c sources, but the discovery commands for actually finding them in the project were missing the extensions (they had every other extension in there).

The fix is in #4469.
